### PR TITLE
Add Microsoft.Quantum.Standard.Visualization to default packages

### PR DESCRIPTION
**Pull request by cgranade**
This PR adds Microsoft.Quantum.Standard.Visualization, a package that provides Jupyter visualization support for standard library operations, to the default packages loaded at IQ# startup.

### CODE REVIEW

File: src/AzureClient/AzureClient.csproj [hunk 1]
```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Quantum.Client" Version="0.12.20081729-beta" />
+    <PackageReference Include="Microsoft.Azure.Quantum.Client" Version="0.12.20072811-beta" />
```
Comment (rmshaffer) [inline src/AzureClient/AzureClient.csproj:16]: Did you mean to downgrade these packages, or is this just a merge resolution issue? (here and throughout the PR)

File: src/ExecutionPathTracer/ExecutionPathTracer.csproj [hunk 2]
```diff
@@ -32,7 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\..\build\assets\qdk-nuget-icon.png" Pack="true" Visible="false" PackagePath="" />
+    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.12.20072811-beta" />
```
Comment (rmshaffer) [inline src/ExecutionPathTracer/ExecutionPathTracer.csproj:35]: Looks like a bad merge here, I think you meant to keep the line with the icon?
```
<None Include="..\..\build\assets\qdk-nuget-icon.png" Pack="true" Visible="false" PackagePath="" />
```

Comment (cgranade) [reply src/ExecutionPathTracer/ExecutionPathTracer.csproj:35]: That was a merge issue, thanks for noticing that. Will fix.

File: src/AzureClient/AzureClient.csproj (hunk 1 above)
Comment (cgranade) [reply src/AzureClient/AzureClient.csproj:16]: This is currently the latest version which has the new package turned on; hopefully build 0.12.2008.1919 should be done soon, though.

**Comment by rmshaffer** [comment]
Unfortunately (?), there's also a test that depends on the number of auto-loaded packages. I'm sure you would have seen that from the CI failure, but just mentioning it for good measure: https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build/results?buildId=13302&view=logs&j=9e36a536-c213-5644-c2ab-401b20f4b829&t=57baf6f6-621f-5161-f318-42bbd94a0177&l=90

**Comment by cgranade** [comment]
> Unfortunately (?), there's also a test that depends on the number of auto-loaded packages. I'm sure you would have seen that from the CI failure, but just mentioning it for good measure: https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build/results?buildId=13302&view=logs&j=9e36a536-c213-5644-c2ab-401b20f4b829&t=57baf6f6-621f-5161-f318-42bbd94a0177&l=90
> 
> Edit: actually, there are two such tests! The other one: https://dev.azure.com/ms-quantum-public/Microsoft%20Quantum%20(public)/_build/results?buildId=13302&view=logs&j=9e36a536-c213-5644-c2ab-401b20f4b829&t=57baf6f6-621f-5161-f318-42bbd94a0177&l=215

Thanks for the heads up, that saves me a lot of time debugging the CI issue! Will update as soon as 0.12.2008.1919 is completed, then. Thanks!

File: src/ExecutionPathTracer/ExecutionPathTracer.csproj (hunk 2 above)
Comment (rmshaffer) [reply src/ExecutionPathTracer/ExecutionPathTracer.csproj:35]: Thanks! And just to clarify, the "bad" in my comment was referring to the suggestion from the merge tool, not to the user of the merge tool... :-)

**Review by rmshaffer — DISMISSED** [review]

**Pull request merged**